docs/scarpet/api/BlocksAndWorldAccess.md

```diff
@@ -241,9 +241,10 @@ If called with no args, returns `'clear'`, `'rain` or `'thunder'` based on the c
 always return `'thunder'`, if not will return `'rain'` or `'clear'` based on the current weather.
 
 With one arg, (either `'clear'`, `'rain` or `'thunder'`), returns the number of remaining ticks for that weather type.
-NB: It can thunder without there being a thunderstorm, there has to be both rain and thunder to form a storm.
+NB: It can thunder without there being a thunderstorm; there has to be both rain and thunder to form a storm. So if
+running `weather()` returns `'thunder'`, you can use `weather('rain')>0` to see if there's a storm going on.
 
-With two args, sets the weather to `type` for `ticks` ticks.
+With two args, sets the weather to that `type` for `ticks` ticks.
```

Comment:
I've seen it without that mostly, and it sounds better for me, but I don't know which is more correct.
```suggestion
With two args, sets the weather to `type` for `ticks` ticks.
```
Or maybe
```suggestion
With two args, sets the weather to the given `type` for `ticks` ticks.
```
?